zellij-server/src/pty.rs

```diff
@@ -237,19 +242,36 @@ impl Pty {
     pub fn spawn_terminals_for_layout(
         &mut self,
         layout: Layout,
-        terminal_action: Option<TerminalAction>,
+        default_shell: Option<TerminalAction>,
     ) {
-        let total_panes = layout.total_terminal_panes();
+        let total_runnables = layout.total_runnables();
```

Comment:
I think calling this method "total_runnables" is a little confusing. "total" implies we're talking about an amount, and here we're getting back a run instruction... maybe we can call it `extract_run_instructions` and then in the loop call the singular version `run_instruction`? What do you think?

Reply:
Thank you that is very helpful. I think that is a good idea.